tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/configurables/ConfigurableTest.java

```diff
@@ -151,6 +152,57 @@ public void testNegativeCasesInProjects(String projectName, String errorMsg) thr
         errorLog.waitForText(5000);
     }
 
+    @Test(dataProvider = "negative-tests")
+    public void testNegativeCases(String tomlFileName, String errorMsg) throws BallerinaTestException {
+        Path projectPath = Paths.get(negativeTestFileLocation, "configProject").toAbsolutePath();
+        Path tomlPath = Paths.get(negativeTestFileLocation, "config_files", tomlFileName  + ".toml").toAbsolutePath();
+        LogLeecher errorLog = new LogLeecher(errorMsg, ERROR);
+        bMainInstance.runMain("run", new String[]{"main"}, addEnvVariables(tomlPath.toString()), new String[]{},
+                new LogLeecher[]{errorLog}, projectPath.toString());
+        errorLog.waitForText(5000);
+    }
+
+    @DataProvider(name = "negative-tests")
+    public Object[][] getNegativeTests() {
+        return new Object[][]{
+                {"no_module_config", "Value not provided for required configurable variable 'main:stringVar'"},
+                {"invalid_org_name", "Value not provided for required configurable variable 'main:stringVar'" },
+                {"invalid_org_structure", "invalid module structure found for module 'testOrg.main'. " +
```

Comment:
lower case `i` need to be fixed

Reply:
This is fixed in PR #28829

Reply:
actually it's the other way around it seems, others need to lower cased